google/cloud/accessapproval_v1/gapic/access_approval_client.py

```diff
@@ -658,16 +628,37 @@ def update_access_approval_settings(
 
                 If a dict is provided, it must be of the same form as the protobuf
                 message :class:`~google.cloud.accessapproval_v1.types.AccessApprovalSettings`
-            update_mask (Union[dict, ~google.cloud.accessapproval_v1.types.FieldMask]): The update mask applies to the settings. Only the top level fields
-                of AccessApprovalSettings (notification_emails & enrolled_services) are
-                supported. For each field, if it is included, the currently stored value
-                will be entirely overwritten with the value of the field passed in this
-                request.
-
-                For the ``FieldMask`` definition, see
-                https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#fieldmask
-                If this field is left unset, only the notification_emails field will be
-                updated.
+            update_mask (Union[dict, ~google.cloud.accessapproval_v1.types.FieldMask]): Protocol Buffers - Google's data interchange format Copyright 2008
+                Google Inc. All rights reserved.
+                https://developers.google.com/protocol-buffers/
+
+                Redistribution and use in source and binary forms, with or without
+                modification, are permitted provided that the following conditions are
+                met:
+
+                ::
+
+                    * Redistributions of source code must retain the above copyright
+
+                notice, this list of conditions and the following disclaimer. \*
+                Redistributions in binary form must reproduce the above copyright
+                notice, this list of conditions and the following disclaimer in the
+                documentation and/or other materials provided with the distribution. \*
+                Neither the name of Google Inc. nor the names of its contributors may be
+                used to endorse or promote products derived from this software without
+                specific prior written permission.
+
+                THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
+                IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED
+                TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A
+                PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER
+                OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+                EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+                PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+                PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+                LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+                NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+                SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```

Comment:
Not sure why the license is showing up here. 😕

Reply:
This docstring appeared in a regen even though the library is using Artman. The problem is likely in protoc-docs-plugin itself.